packages/@okta/vuepress-theme-prose/components/Home.vue

```diff
@@ -5,6 +5,31 @@
         <HomeSidebar />
       </div>
       <div class="col-xl-10 col-lg-10 col-md-12 col-sm-12">
+        <div class="okta-game-banner row">
+          <div class="okta-game-banner--logo">
+            <img src="/img/okta-game/okta-game-logo.svg" />
+          </div>
+          <div class="okta-game-banner--image">
+            <img src="/img/okta-game/okta-game-image.svg" />
+          </div>
+          <div class="okta-game-banner--details">
+            <p>Can you turn this small business into a global powerhouse?</p>
+            <div class="okta-game-banner--details-buttons">
+              <SmartLink
+                :item="{ link: 'https://codetycoongame.com/' }"
+                classes="play-now-btn"
+              >
+                Play Now
```

Comment:
This button is a little bit uncentered, can you see if it can be centered better? Also curious if we can use the same button class as in the other section that was added before since it's the same color/style.